app/assets/javascripts/discourse/app/lib/click-track.js

```diff
@@ -9,9 +9,18 @@ import { isTesting } from "discourse-common/config/environment";
 import discourseLater from "discourse-common/lib/later";
 import { selectedText } from "discourse/lib/utilities";
 import { wantsNewWindow } from "discourse/lib/intercept-click";
+import deprecated from "discourse-common/lib/deprecated";
 
-export function isValidLink($link) {
-  const link = $link[0];
+export function isValidLink(link) {
+  // eslint-disable-next-line no-undef
+  if (link instanceof jQuery) {
+    link = link[0];
+
+    deprecated("isValidLink now expects an Element, not a jQuery object", {
+      since: "2.9.0.beta7",
+      dropFrom: "2.10.0.beta1",
```

Comment:
`3.0.0`

(internal `/t/3663/20`)

Reply:
I'm thinking about dropping `dropFrom`. It's not like we're actually going to remove it after the next release. 😛